index.Rmd

```diff
@@ -78,7 +78,7 @@ summary(fit)
 ```
 
 
-# Results
+# Results 
```

Comment:
The interactivity of the first plot is a great touch, as it allows you to see specific opoid related deaths in each county upon hovering over the points.  The plot is also very busy with all NY counties represented and often times overlapping and with very similar coloring.  Maybe breaking this up into several plots, each with several counties grouped by region would help to get rid of some of the clutter.  Or maybe using facet_wrap(), but that would also create a large number of plots that couldn't be compared to each other as easily.